manager/filters.py

```diff
@@ -194,13 +222,42 @@ def _filter_system_by_opt_out(query, args):
         object: Modified query with system opt out filter applied
     """
     # pylint: disable=singleton-comparison
+    if args['excluded'] is not None:
+        if args['opt_out'] is None:
+            return query
+        return query.where(SystemPlatform.opt_out == True)  # noqa: E712
```

Comment:
this doesn't seem correct, it sets SystemPlatform.opt_out == True for excluded=true,false and opt_out=false combination

Reply:
Please read the API documentation. As there is no easy way how to make opt_out and exclude work together I've decided that it would be better to reset to default behavior if both of them are entered.
This is temporary solution once we remove opt_out filtering parameter in the future.

Reply:
Yes but what's default behavior about turniing opt_out on when both excluded and opt_out says otherwise?